go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/SafetyCulture/safetyculture-exporter
 
-go 1.22
+go 1.23
 
-toolchain go1.22.3
+toolchain go1.24.1
```

Comment:
is it okay to use a different toolchain version?

Reply:
that's automatically by go. I cannot touch that. Go will put it back

Reply:
yes